src/services/networkService.js

```diff
@@ -190,10 +189,13 @@ class NetworkService {
         return false;
       });
 
+    const blockGetterHeight = services_synced_heights.find(i => i.service === 'block_getter' ).height;
+    const watcherSynced = last_seen_eth_block_number - blockGetterHeight <= config.checkSyncInterval;
```

Comment:
was wondering if the block heights will ever become so large they become unsafe ints, but i think thats a long ways off

Reply:
yeah, 😬 . It should be fine though, even with current block heights on Mainnet, everything indicates that we should be good to use regular integer for a while